bin/elastic-server.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -xe
+manage.sh onStart #|| exit $?
```

Comment:
How would a "real" init system change this? I'm not sure what you mean by real init though.